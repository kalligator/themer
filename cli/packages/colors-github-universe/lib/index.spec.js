const { colors } = require('./index');

describe('themer color set', () => {
  it('should provide complete dark color set', () => {
    expect(colors).toMatchSnapshot();
  });
});
