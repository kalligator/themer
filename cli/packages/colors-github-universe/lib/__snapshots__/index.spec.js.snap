// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`themer color set should provide complete dark color set 1`] = `
Object {
  "dark": Object {
    "accent0": "#EF4E7C",
    "accent1": "#F37055",
    "accent2": "#F79532",
    "accent3": "#6EBB82",
    "accent4": "#09B399",
    "accent5": "#1299AD",
    "accent6": "#5073B8",
    "accent7": "#A067AB",
    "shade0": "#23292D",
    "shade7": "#FDFDFD",
  },
}
`;
